components/member_list_team/member_list_team.tsx

```diff
@@ -41,8 +53,8 @@ export default class MemberListTeam extends React.Component {
     }
 
     componentDidMount() {
-        this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId).then(({data}) => {
-            if (data) {
+        (this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId) as any).then((result: any) => {
```

Comment:
Is it possible to specify the actual type here instead of using any? I'm not completely sure if that's possible when these actions are coming from a non-TypeScript file though.

Reply:
`searchProfiles` and `getTeamStats` can be defined by its types instead of importing them. Still some problems with ActionFunc, since it can return {data} or {error}, and therefore making some extra checks needed and potentially the use of `any`.

Regarding the type, here we can define the type, but there is a big difference between the inferred type (the one that will be used) and the actual type it has. The behaviour of these functions is pretty similar to the actions in redux, and the shape is almost the same. But the inferred type yields `() => () => Promise` instead of `() => Promise`, and it seems there is no way to make them work. I tried to port also these functions, but was not successful. Nevertheless, I can try again if you deem it needed.